src/webgl/p5.Shader.js

```diff
@@ -578,6 +578,11 @@ p5.Shader = class {
           stride || 0,
           offset || 0
         );
+        if (divisor !== undefined) {
+          this._renderer.GL.vertexAttribDivisor(loc, divisor);
+        } else if (this._renderer.webglVersion === constants.WEBGL2) {
+          this._renderer.GL.vertexAttribDivisor(loc, 0);
```

Comment:
Not sure I understand what this divisor is doing, though I assume it's ok to have it be zero?

Reply:
The divisor is how many instances get the same attribute value in a row before moving on to the next value in the buffer. The default behavior is the same as 0, which means don't repeat anything, just keep pulling new values from the buffer like normal. For attributes that we want to repeat, we call set a different divisor to reuse the value for the whole instance or for multiple instances.

I can also add something like this in a comment explaining what it is so that it's less necessary to have a bunch of MDN tabs open to understand what the code is trying to do 😅

Reply:
That makes sense to me, thanks for the explanation!